bot.go

```diff
@@ -1,10 +1,61 @@
 package main
 
-import "fmt"
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
 
-// import "github.com/bwmarrin/discordgo"
-// import "github.com/go-co-op/gocron/v2"
+	"github.com/bwmarrin/discordgo"
+	// "github.com/go-co-op/gocron/v2"
+)
+
+var (
+	BotToken = flag.String("token", "", "Bot Token")
+)
+
+func init() { flag.Parse() }
 
 func main() {
-	fmt.Print("arst")
+
+	if *BotToken == "" {
+		fmt.Println("Not token specified")
+		return
+	}
+
+	dg, err := discordgo.New("Bot " + *BotToken)
+	if err != nil {
+		fmt.Println("YOU RUINED IT: ", err)
+		return
+	}
+
+	dg.AddHandler(func(s *discordgo.Session, event *discordgo.Ready) {
+		s.ChannelMessageSend("522472817283956745", "FARTS WE GO IN")
+	})
+	dg.AddHandler(messageCreate)
+
+	// Receive message events
+	// THIS IS DOGSHIT
```

Comment:
this comment is not very good. Could you please revise it

Reply:
no